src/verilog/verilog_lowering.cpp

```diff
@@ -152,8 +159,136 @@ exprt to_bitvector(const exprt &src)
   }
 }
 
+exprt verilog_lowering_system_function(const function_call_exprt &call)
+{
+  auto identifier = to_symbol_expr(call.function()).get_identifier();
+  auto &arguments = call.arguments();
+
+  if(identifier == "$signed" || identifier == "$unsigned")
+  {
+    // lower to typecast
+    DATA_INVARIANT(
+      arguments.size() == 1, id2string(identifier) + " must have one argument");
```

Comment:
```suggestion
      arguments.size() == 1, id2string(identifier) + " takes one argument");
```
for consistency with the other cases.